mac/Keyman4MacIM/make-km-dmg.sh

```diff
@@ -139,6 +139,8 @@ elif [[ ! -d "$DEST_DIR" ]]; then
 	fail "Destination dir exists but is not a directory: $2"
 fi
 
+# TODO: Check that no Keyman volume is already mounted.
```

Comment:
I assume leaving this a `# TODO` for now is intentional?

Reply:
yes; I am expecting that we'll stop using .dmg in the future in preference for .pkg, so this won't be necessary to do anyway.